src/datumaro/plugins/data_formats/coco/page_mapper.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import struct
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Callable, Dict, Iterator, List, Tuple
+
+from datumaro.util import parse_json
+
+__all__ = ["COCOPageMapper"]
+
+
+@dataclass
+class AnnotationPageMap:
+    offsets: List[int] = field(default_factory=list)
+    sizes: List[int] = field(default_factory=list)
+    prev_pts: List[int] = field(default_factory=list)
+
+
+@dataclass
+class ItemPage:
+    offset: int
+    size: int
+    ann_last_pt: int
+
+
+@dataclass
+class BraceStatus:
+    flush: bool = False
+    level: int = 0
+    get_started: bool = False
+
+
+@dataclass
+class CurlyStatus:
```

Comment:
Could you figure out what `BraceStatus` and `CurlyStatus` mean?

Reply:
https://github.com/openvinotoolkit/datumaro/pull/1091/commits/ab88cf9b2462dbd8976b25bb810931cab273ba46
rename `BraceStatus -> BracketStatus` and `CurlyStatus -> BraceStatus`. It also adds docstrings to them.